docs/quickstart/install-and-use-a-package-in-visual-studio.md

```diff
@@ -9,7 +9,7 @@ ms.topic: quickstart
 
 # Quickstart: Install and use a package in Visual Studio (Windows only)
 
-NuGet packages contain reusable code that other developers make available to you for use in your projects. See [What is NuGet?](../What-is-NuGet.md) for background. Packages are installed into a Visual Studio project using the NuGet Package Manager or the Package Manager Console. This article demonstrates the process using the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package and a Windows Presentation Foundation (WPF) project. The same process applies to any other .NET or .NET Core project.
+NuGet packages contain reusable code that other developers make available to you for use in your projects. See [What is NuGet?](../What-is-NuGet.md) for background. Packages are installed into a Visual Studio project using the NuGet Package Manager or the Package Manager Console. You can also install nuget packages via the [dotnet command line](install-and-use-a-package-using-the-dotnet-cli.md). This article demonstrates the process using the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package and a Windows Presentation Foundation (WPF) project. The same process applies to any other .NET or .NET Core project.
```

Comment:
What do you think of merging your new sentence into the previous one? Something like:

> ... Packages are installed into a Visual Studio project using the [dotnet command line](install-and-use-a-package-using-the-dotnet-cli.md), the NuGet Package Manager, or the Package Manager Console. This article demonstrates ...